vendor/assets/javascripts/better_markdown.js

```diff
@@ -335,7 +335,7 @@
       // __foo__ is reserved and not a pattern
       if ( i.match( /^__.*__$/) )
         continue;
-      var l = i.replace( /([\\.*+?^$|()\[\]{}])/g, "\\$1" )
+      var l = i.replace( /([\/\\.*+?^$|()\[\]{}])/g, "\\$1" )
```

Comment:
I am confused by this change, and worried that you are adding semantics that can easily break elsewhere in the parser

We have found that hacking on the md parser is probably the most complicated and error prone part of hacking on Discourse and this may break stuff like `\*hello*` I don't know.